docs/_static/vcr/v2.yaml

```diff
@@ -0,0 +1,245 @@
+openapi: "3.0.0"
+info:
+  title: Nuts Verifiable Credential API spec
+  description: API specification for Verifiable Credentials
+  version: 1.0.0
+  license:
+    name: GPLv3
+servers:
+  - url: http://localhost:1323
+paths:
+  /internal/vcr/v2/issuer/vc:
+    post:
+      summary: Creates a new Verifiable Credential
+      description: |
+        Creates a new Verifiable Credential.
+        
+        error returns:
+        * 400 - One or more of the given parameters are invalid
+        * 500 - An error occurred while processing the request
+      operationId: "issueVC"
+      tags:
+        - credential
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IssueVCRequest'
+      responses:
+        "200":
+          description: "New VC has been created successfully. Returns the Verifiable Credential."
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/VerifiableCredential'
+        default:
+          $ref: '../common/error_response.yaml'
+
+  /internal/vcr/v2/issuer/vc/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: URL encoded ID.
+        required: true
+        example:
+          - "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467"
+        schema:
+          type: string
+    get:
+      summary: "Resolves an issued verifiable credential"
+      description: >
+        The resolutionResult contains the credential regardless of the validity.
+        The resolutionResult.currentStatus must be checked to see if the credential is valid.
+
+        error returns:
+        * 404 - Corresponding credential could not be found
+        * 500 - An error occurred while processing the request
+      operationId: "resolveIssuedVC"
+      parameters:
+        - name: resolveTime
+          in: query
+          description: a rfc3339 time string for resolving a VC at a specific moment in time
+          example: "2012-01-02T12:00:00Z"
+          schema:
+            type: string
+      tags:
+        - credential
+      responses:
+        "200":
+          description: Credential has been found and is returned.
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/ResolutionResult'
+        default:
+          $ref: '../common/error_response.yaml'
+    delete:
+      summary: "Revoke an issued credential"
+      description: |
+        Revoke a credential.
+
+        error returns:
+        * 400 - Credential can't be revoked. Most likely due to a missing private key.
+        * 404 - Credential is not found
+        * 409 - Credential has already been revoked
```

Comment:
did we already do this? Because revoking twice shouldn't really make a difference?

Reply:
Seems strange to allow a revocation twice. This will publish the transaction again. I don't see any advantages to that over checking for existing revocation and failing if already revoked.

Reply:
It doesn't hurt and is safe to do it twice. But it's a client error since the calling software obvious has a bug (or poor design)